app/react/Templates/components/PropertyOption.js

```diff
@@ -15,7 +15,9 @@ export class PropertyOption extends Component {
   }
 
   addProperty() {
-    this.props.addProperty({ label: this.props.label, type: this.props.type });
+    if (!this.props.disabled) {
```

Comment:
Please ensure that the ESLINT plugin is working for your IDE, this should not be committed with this warnings.